docs/Installation.md

```diff
@@ -89,7 +89,7 @@ sudo dnf -y install jq perl-Class-Method-Modifiers perl-Config-IniFiles perl-DBD
 Install dependencies not available from binary packages:
 
 ```sh
-sudo cpanm Daemon::Control JSON::Validator Log::Any Log::Any::Adapter::Dispatch Starman Plack::Middleware::ReverseProxy

```

Comment:
There is no package `perl-Plack-Middleware-ReverseProxy` available for CentOS 7.

Reply:
> There is no package `perl-Plack-Middleware-ReverseProxy` available for CentOS 7.

But available from CPAN?

Reply:
Okay, I'll revert that change.

Reply:
In Engine a separate section for CentOS 7 has been proposed in https://github.com/zonemaster/zonemaster-engine/pull/1080. I suggest that we do the same thing here to make it as good as possible for Rocky Linux.

Reply:
The installation document for Backend is quite long and heavy, I would suggest not to add an extra section specific to CentOS 7 since the installation works with the instructions from the Rocky Linux section if you use `yum` instead of `dnf`.

Reply:
Installing five or six distributions from CPAN instead isn't a big difference.

Reply:
> The installation document for Backend is quite long and heavy, I would suggest not to add an extra section specific to CentOS 7 since the installation works with the instructions from the Rocky Linux section if you use `yum` instead of `dnf`.

Then, at least that information should be stated. And there should be a headline (entrance) for CentOS Linux 7. Furthermore, PostgreSQL is not support for CentOS Linux 7.

Reply:
I'll leave it to @pnax to make the CentOS updates.